settings.gradle.kts

```diff
@@ -144,7 +144,7 @@ dependencyResolutionManagement {
       version("netty-version", "4.1.66.Final")
       version("protobuf-java-version", "3.19.4")
       version("slf4j-version", "2.0.3")
-      version("swirlds-version", "0.38.0-adhoc.xcca9daa7")
+      version("swirlds-version", "0.38.0-adhoc.xa85f09f3")
```

Comment:
I am not sure that we can use this version in development. we need first to merge the platform into there devlop and regenerate the version from devlop and than update it.

Reply:
Yes, I knew. Not sure what to put there yet. Wait for platform new build to put there.

Reply:
maybe let's merge without it and then we can change this tag and update it on develop in services.

Reply:
Without change, code won't compile,